src/software/world/robot_test.cpp

```diff
@@ -304,35 +304,38 @@ TEST_F(RobotTest, get_timestamp_history)
     EXPECT_EQ(prevTimestamps, previous_timestamps);
 }
 
-TEST_F(RobotTest, get_capabilities_blacklist)
+TEST_F(RobotTest, get_unavailable_capabilities)
 {
-    std::set<RobotCapability> blacklist = {
+    std::set<RobotCapability> unavailableCapabilities = {
         RobotCapability::Dribble,
         RobotCapability::Chip,
     };
 
-    Robot robot = Robot(0, Point(3, 1.2), Vector(-3, 1), Angle::fromDegrees(0),
-                        AngularVelocity::fromDegrees(25), current_time, 3, blacklist);
+    Robot robot =
+        Robot(0, Point(3, 1.2), Vector(-3, 1), Angle::fromDegrees(0),
+              AngularVelocity::fromDegrees(25), current_time, 3, unavailableCapabilities);
 
-    EXPECT_EQ(blacklist, robot.getCapabilitiesBlacklist());
+    EXPECT_EQ(unavailableCapabilities, robot.getUnavailableCapabilities());
 }
 
-TEST_F(RobotTest, get_capabilities_whitelist)
+TEST_F(RobotTest, get_available_capabilities)
 {
-    std::set<RobotCapability> blacklist = {
+    std::set<RobotCapability> unavailableCapabilities = {
         RobotCapability::Dribble,
         RobotCapability::Chip,
     };
 
-    Robot robot = Robot(0, Point(3, 1.2), Vector(-3, 1), Angle::fromDegrees(0),
-                        AngularVelocity::fromDegrees(25), current_time, 3, blacklist);
+    Robot robot =
+        Robot(0, Point(3, 1.2), Vector(-3, 1), Angle::fromDegrees(0),
+              AngularVelocity::fromDegrees(25), current_time, 3, unavailableCapabilities);
 
-    // whitelist = all capabilities - blacklist
+    // possible capabilities = all capabilities - unavailableCapabilities
```

Comment:
possible capabilities -> available capabilities to keep it consistent with the rest of the phrasing

Reply:
Fixed!